datatype/custom.go

```diff
@@ -20,62 +20,50 @@ import (
 	"io"
 )
 
-type CustomType interface {
-	DataType
-	GetClassName() string
+// CustomType is a data type that represents a CQL custom type.
+// +k8s:deepcopy-gen=true
+// +k8s:deepcopy-gen:interfaces=github.com/datastax/go-cassandra-native-protocol/datatype.DataType
+type CustomType struct {
```

Comment:
In this package, some interfaces are now structs. This is because the tool cannot generate deepcopy methods for unexported structs.

But in fact, I don't even know why I created these interfaces in the first place. The only impl is the associated struct, so they seem pretty overkill.

Reply:
Also note: the names are going to change with #53.